src/test/java/com/baidu/hugegraph/api/traverser/KoutApiTest.java

```diff
@@ -110,10 +110,10 @@ public void testKoutGetBothAllWithCapacity() {
 
         Assert.assertThrows(ServerException.class, () -> {
             koutAPI.get(markoId, Direction.BOTH, null,
-                        2, false, -1L, -1L, 1L);
+                        2, false, -1L, 1L, 2L);
         }, e -> {
             String expect = "Capacity can't be less than limit, " +
-                    "but got capacity '1' and limit '-1'";
+                    "but got capacity '1' and limit '2'";
             Assert.assertContains(expect, e.getMessage());
         });
     }
```

Comment:
add one test with capacity 1 and limit -1 to ensure limit can't be NO_LIMIT when capacity is not NO_LIMIT